config/config.js

```diff
@@ -14,6 +14,7 @@ module.exports = {
     database: process.env.POSTGRES_DB,
     host: process.env.POSTGRES_HOST,
     dialect: 'postgres',
+    logging: false,
```

Comment:
Whoops, this slipped in, I can take this out next time I add a feature. I don't think anyone will really miss the default of extra verbose sequelize log statements in the meantime.